docs/guide/controller-programming.md

```diff
@@ -98,11 +233,47 @@ The call to the `wb_distance_sensor_get_value` function retrieves the latest val
 
 Note that some device return vector values instead of scalar values, for example these functions:
 
+%tab-component "language"
+%tab "C"
 ```c
 const double *wb_gps_get_values(WbDeviceTag tag);
 const double *wb_accelerometer_get_values(WbDeviceTag tag);
 const double *wb_gyro_get_values(WbDeviceTag tag);
 ```
+%tab-end
+
+%tab "C++"
+```cpp
+webots::GPS::const double *getValues() const;
+webots::Accelerometer::const double *getValues() const;
+webots::Gyro::const double *getValues() const;
+```
+%tab-end
+
+%tab "Python"
+```python
+[x, y, z] GPS.getValues()
```

Comment:
I didn't find explicit documentation about how to present methods in class within one line for Python and Java, is my guess right ?

Reply:
There is no such convention in Python for the return value. It is usually explained in the description text. Maybe our Python sections should contain the source code of the Python method as usually documented in Python, but also a sentence explaining the return value as it is the case in the official Python documentation.

Reply:
So actually it is displayed like you suggested, but it's not really esthetic since the text drops down and disappears as we change from a tab to another. Even if it belongs to the right place, maybe I can suggest to leave it in the tab itself under a comment so it will remain in the "visual tab" and not at the beginning of the following explanation, almost unnoticeable.

Or we can add a new line before the generic explanation text so we really see this line:
`Returns the sensor measurement as an array of 3 floating point numbers: [x, y, z].` is belonging to the tab and not the text...

Reply:
Yes you idea of adding it as a comment in the tab seems very good. Let's go ahead with that.